core/module.cc

```diff
@@ -213,24 +213,19 @@ void Module::ProcessBatch(bess::PacketBatch *) {
 }
 
 task_id_t Module::RegisterTask(void *arg) {
-  task_id_t id;
-  struct task *t;
+  Worker *w = get_next_active_worker();
+  auto *c = w->s()->default_leaf_class();
```

Comment:
Example of the auto confusion -- is this the worker class type, or is this an actual worker instance?